Server/User/services/booking_notification_service.py

```
@@ -6,6 +6,116 @@
+    html_part = MIMEText(html_content, 'html')
+    msg.attach(html_part)
+
+    # Send email
+    try:
+        with smtplib.SMTP(settings.smtp_host, settings.smtp_port) as server:
+            server.starttls()
+            server.login(settings.smtp_email, settings.smtp_password)
+            server.sendmail(settings.smtp_email, [recipient_email], msg.as_string())
+
+        print(f"[booking-cancellation] Email sent successfully to {recipient_email}")
+        return True
```

Ruff (0.13.3)

Consider moving this statement to an `else` block

(TRY300)